src/__snapshots__/shared-render.test.js.snap

```diff
@@ -1,23 +1,47 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`deep: output matches snapshot 1`] = `
-VNode {
-  "attributes": undefined,
-  "children": Array [
-    "second first",
-  ],
-  "key": undefined,
-  "nodeName": "div",
-}
+exports[`deep: context matches snapshot 1`] = `
+preact-render-spy (1 nodes)
+-------
+<div>second first</div>
+
+`;
+
+exports[`deep: find matches snapshot 1`] = `
```

Comment:
We only get the 3 spans because: 

```
expect(context.find('span')).toMatchSnapshot();
```
so the parent `div` isn't a part of the snapshot.

if we snapshotted `expect(context).toMatchSnapshot()` as well we get:

```
exports[`deep: find matches snapshot 2`] = `
preact-render-spy (1 nodes)
-------
<div>
  <span onClick={[Function onClick]}>click</span>
  <span attr="text attr">text</span>
  <span
    itsTrue={true}
    itsFalse={false}
  >
    bools
  </span>
</div>

`;
```